proxy/src/http/org.rs

```diff
@@ -250,6 +281,30 @@ mod handler {
 
         Ok(reply::with_status(reply::json(&tx), StatusCode::CREATED))
     }
+
+    /// Register a member under an org on the Registry.
+    pub async fn register_member<R: registry::Client>(
+        registry: http::Shared<R>,
+        subscriptions: notification::Subscriptions,
+        id: String,
+        input: super::RegisterMemberInput,
+    ) -> Result<impl Reply, Rejection> {
+        // TODO(xla): Get keypair from persistent storage.
+        let fake_pair = radicle_registry_client::ed25519::Pair::from_legacy_string("//Alice", None);
```

Comment:
Just reading through this to get more familiar with proxy - is this keypair specific to the user and necessary for authenticating every transaction on the registry?

Reply:
Once a user handle is registered with a specific author key, this key will be associated with the user on the registry. Not only does every transaction has to be authored with a key, some transaction require the association with the user handle to check for permissions to do the transaction (eg registering a member under an org can only be done by a key associated with a user who is a member of the org)